packages/less/test/lessc-alpha.mjs

```
@@ -0,0 +1,228 @@
+    assert.doesNotMatch(failureStderr, /^Error: Less parser error\.$/m,
+        'lessc must not append a duplicate plain Error after a Linecraft diagnostic');
+
+    const silentFailure = await runLessc(['--silent', broken]);
+    assert.equal(silentFailure.code, 1, '--silent should still fail malformed input');
+    assert.equal(silentFailure.stdout, '');
+    assert.equal(silentFailure.stderr, '', '--silent must suppress Jess diagnostics');
+
+    const noColorFailure = await runLessc(['--no-color', broken]);
+    assert.equal(noColorFailure.code, 1, '--no-color should still fail malformed input');
+    assert.equal(noColorFailure.stdout, '');
+    assert.doesNotMatch(noColorFailure.stderr, /\x1B/u,
```

ESLint

[error] Unexpected control character(s) in regular expression: \x1b.

(no-control-regex)